.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "tools/uf2"]
 	path = tools/uf2
 	url = https://github.com/Microsoft/uf2.git
+[submodule "atmel-samd/frozen/Adafruit_CircuitPython_NeoPixel"]
+	path = atmel-samd/frozen/Adafruit_CircuitPython_NeoPixel
```

Comment:
Good point. I didn't do that initially because `FROZEN_MPY_DIRS` isn't implemented in all the ports. But it will be implemented in most or all of our ports eventually.

Changed and tested on a CPX.